components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/resources/devportal-api.yaml

```diff
@@ -2972,7 +2972,7 @@ paths:
         Get the List of API Developer Portal Alert Types.
       description: |
         This operation is used to get the list of supportd alert types for the 'subscriber' agent.
-      operationId: getStoreAlertTypes
+      operationId: getDevportalAlertTypes
```

Comment:
When it is used inside methods etc, shall we use DevPortal -- here eg: getDevPortalAlertTypes

Reply:
Fixed in 7033c14